_posts/2019-09-09-code-school-made-me-a-better-pm.md

```diff
@@ -0,0 +1,67 @@
+drupal_planet_summary: |
+  "How learning the basics of the command line, git, and frontend languages made me a better advocate for clients, development teams, and a better project manager."
+description: "How learning the basics of the command line, git, and frontend languages made me a better advocate for clients, development teams, and a better project manager."
+image: "/assets/img/blog/ancient-language.jpg"
+featured_image: "/blog/ancient-language.jpg"
+featured_image_alt: "Ancient written language."
+
+---
+
+Fun fact: before transitioning to the Digital Project Manager career, I spent ~6 months out of the year traveling internationally for work (for an amazing organization called [Engineering World Health](http://www.ewh.org) -- yes, I’m biased, but check it out!).
+
+This meant I spent a lot of time in countries in which I did not speak the local language (including Kinyarwanda, Swahili, Khmer, Nepali, and Mongolian). Before each 2-3 month trip, I’d learn some basic phrases (you know, the classics: “Hello,” “Where is the bathroom?”, and “Please, stop the bus, I need to get out!”). However, I found that for each new language, it took about a week for my ears to adjust to the new sounds and patterns so that I could reliably pick up words and phrases.
+
+
+<div class="blog-image-full-width">
+<img alt="Map depicting Maddy's travels" src="/assets/img/blog/maddy-travels-map.png">
+<span class="caption">Those tiny red dots are areas I’ve lived, worked, and traveled over the past 3 years.</span>
+</div>
+
+I’ve found that the same principle applies to working in web development. In my last blog post, I wrote about [Learning to Speak Drupal](https://savaslabs.com/2019/04/22/top-5-things-to-know-to-speak-drupal.html). Now, I’m reflecting on how learning the basics of the command line, git, and frontend languages made me a better advocate for clients, development teams, and a better project manager as a whole.
+
+Since I’m both perpetually curious and respectful of my coworker’s time, I looked to a free, open-source code school called The Odin Project to begin learning the “basic phrases” of web development. [The Odin Project](https://www.theodinproject.com) teaches Ruby on Rails as a backend language and framework, while Savas works primarily with PHP/Drupal. However, I still found it to be a fantastic roadmap for learning the basics.
+
+I started with the [Web Development 101](https://www.theodinproject.com/courses/web-development-101) course and would recommend it to anyone who prefers learning by doing (and failing and then doing again…). While working my way through the course, I built this groovy [etch-a-sketch](https://maddybvh.github.io/etch-a-sketch/).
+
+<div class="blog-image-large">
+<img alt="Yoshi depicted on virtual etch-a-sketch" src="/assets/img/blog/etch-a-sketch.png">
+<span class="caption">[Ro](https://savaslabs.com/company/ro-wang/), a senior PM at Savas, used my etch-a-sketch to draw a killer rendition of Yoshi</span>
```

Comment:
You'll need to correct this as the Markdown shows up on the post. I believe the rule is when you use HTML on a given line, it won't do the Markdown formatting on that same line, so you need to do it all in HTML.